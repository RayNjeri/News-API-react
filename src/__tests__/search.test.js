import React from 'react';
import { mount, shallow } from 'enzyme';
import shallowToJSON from 'enzyme-to-json';
import ReactTestUtils from 'react-dom/test-utils';
import Search from '../components/Sourcesearch';

describe('Search component', () => {
    it('Should have a search button', () => {
        const search = shallow(<Search />);
        const tree = shallowToJSON(search);
        expect(tree).toMatchSnapshot();
    });
    it("trigger onSearch correctly", () => {
        const handleSearch = jest.fn();
    });
});