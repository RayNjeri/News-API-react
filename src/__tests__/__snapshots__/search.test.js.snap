// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search component Should have a search button 1`] = `
<div
  className="col-md-2"
  id="search"
>
  <input
    className="searchbox"
    onChange={[Function]}
    placeholder="Source Name"
    style={
      Object {
        "backgroundColor": "#d9d9d9",
      }
    }
    type="text"
    value=""
  />
</div>
`;
